Bottom: 5c67a82f8e0257a0667fc7522b76b071d3739ff4
Top:    3473643aac76e2dcc28438970dfa8f62aac1471e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 20:07:28 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 795b9fe..c4b314e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
